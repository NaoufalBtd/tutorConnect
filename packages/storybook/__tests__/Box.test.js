import React from 'react';
import renderer from 'react-test-renderer';
// import { storiesOf } from '@storybook/react';
// import { action } from '@storybook/addon-actions';
import { render } from '@testing-library/react-native';

import BoxStory from '../src/components/Layout/Box/Box';
import Wrapper from '../src/components/Wrapper';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Wrapper>
        <BoxStory />
      </Wrapper>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
