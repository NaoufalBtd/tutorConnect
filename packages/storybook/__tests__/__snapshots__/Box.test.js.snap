// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-view-175oi2r"
  data-style="1mk50wr-1q64x7k 14bja7p-1q64x7k n7aciq-1q64x7k"
  dir={null}
>
  <div
    className="css-view-175oi2r"
    data-style="y9czwk-1hh0pc0 14bja7p-1q64x7k yljpgc-5lm9f0"
    dir={null}
  >
    <div
      className="css-view-175oi2r"
      data-style="1mk50wr-1q64x7k 14bja7p-1q64x7k aj8jl8-2q7xs9"
      dir={null}
    >
      <div
        className="css-text-1rynq56"
        data-style="1mo3ip0-13uhtwo 1mo3ip0-1j1xgf6 1mo3ip0-1j1xgf6 14bja7p-1q64x7k 12ukb3i-v2qyhg"
        dir="auto"
      >
        BOX
      </div>
    </div>
  </div>
</div>
`;
